firebase.json

```diff
@@ -5,6 +5,18 @@
   },
   "hosting": {
     "public": "web-ng/dist/web-ng",
+    "redirects": [
+      {
+        "source": "/p/[:]new",
```

Comment:
I believe we should use `"source": "/p{,/[:]new}"`. It worked for me. Reference - [redirects](https://firebase.google.com/docs/hosting/full-config#redirects)

Reply:
Just tested it the original change again on our hosted instance and it works. `/p{,/[:]new}` would match both `/p` and `/p/:new`, which is not what we want here.